### PR TITLE
Om feature display journey

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- Creates `Journey` component
- Styles `Journey` component
### Where should the reviewer start?

- src/componentsJourney/Journey.css
- src/components/Journey/Journey.tsx

### How should this be manually tested?
Run `npm start ` and on the browser the Journey component

### What are the relevant tickets?
closes #20  

### Screenshots (if appropriate)
Screen Shot 2021-01-06 at 7 51 18 PM

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...className='title-2'>Dream Home</h1>\n          {/* <h2>Journey Edition</h2> */}\n        </div>\n\n        <div className=\"img-container\">\n          <img src={img} alt=\"\" className='main-image'/>\n        </div>\n\n      </div>\n\n    \n      <div>\n      </div>\n    </section>\n  )\n}\n\nexport default Home;\n","/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/types.tsx",[],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/apiCalls.tsx",[],{"ruleId":"42","replacedBy":"43"},{"ruleId":"44","replacedBy":"45"},{"ruleId":"46","severity":1,"message":"47","line":1,"column":26,"nodeType":"48","messageId":"49","endLine":1,"endColumn":36},{"ruleId":"46","severity":1,"message":"50","line":1,"column":17,"nodeType":"48","messageId":"49","endLine":1,"endColumn":25},{"ruleId":"46","severity":1,"message":"51","line":1,"column":27,"nodeType":"48","messageId":"49","endLine":1,"endColumn":36},"no-native-reassign",["52"],"no-negated-in-lhs",["53"],"@typescript-eslint/no-unused-vars","'useContext' is defined but never used.","Identifier","unusedVar","'useState' is defined but never used.","'useEffect' is defined but never used.","no-global-assign","no-unsafe-negation"]
```
Comment (priyapower) [inline .eslintcache:1]: I believe this is an autogenerated cache file from React

File: public/index.html [hunk 2]
```diff
@@ -10,34 +10,13 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+
```
Comment (priyapower) [inline public/index.html]: Is this file an "error handling pathway" for users who don't have JavaScript enabled?

File: src/_variables.css [hunk 3]
```diff
@@ -4,8 +4,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&family=Gravitas+One&display=swap');
 
 :root{
-  --background_color: ##F0C808;
-  --black_text: ##1A1A1A;
+  --background_color: #F0C808;
```
Comment (priyapower) [inline src/_variables.css:7]: Is this file creating CSS files usable by any part of the app?

File: src/components/App/App.tsx [hunk 4]
```diff
@@ -3,12 +3,12 @@ import {QuestionContext, AllQuestionFormat} from '../../types'
 import {getQuestions} from '../../apiCalls'
 
 //components
+import NavBar from '../NavBar/NavBar';
```
Comment (priyapower) [inline src/components/App/App.tsx:6]: This file holds functionality for the landing page and also, will import the Survey Questions from BE Server

File: src/components/Home/Home.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
+.home-section{
```
Comment (priyapower) [inline src/components/Home/Home.css:1]: CSS file for Home Page

File: src/components/Home/Home.tsx [hunk 6]
```diff
@@ -1,20 +1,27 @@
 import React, { useState, useEffect } from 'react';
-
+import bkg_img from '../../images/home/Big Shoes - Standing Pose.png'
```
Comment (priyapower) [inline src/components/Home/Home.tsx:2]: This file contains HTML for the Landing Page

File: src/components/Journey/Journey.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
+.journey-section{
```
Comment (priyapower) [inline src/components/Journey/Journey.css:1]: Holds CSS for Journey file

File: src/components/Journey/Journey.tsx [hunk 8]
```diff
@@ -1,21 +1,41 @@
 
 import React, { useState, useEffect } from 'react';
+import './Journey.css'
```
Comment (priyapower) [inline src/components/Journey/Journey.tsx:3]: This file holds the HTML for the Journey page

File: src/components/NavBar/NavBar.css [hunk 9]
```diff
@@ -1,7 +1,7 @@
 @import '../../_variables.css';
 
 .nav-container{
-  height: 20vh;
+  height: 15vh;
```
Comment (priyapower) [inline src/components/NavBar/NavBar.css:4]: Holds css for navbar

**Review by priyapower — COMMENTED** [review]
Looks great!

File: .eslintcache (hunk 1 above)
Comment (Atos20) [reply .eslintcache:1]: it is also associated with the Typescript library and started to become an issue with the latest `react-script` upgrade, we are using version 4.0.1 and I tried to downgrade to `4.0.0`  but no luck, I also tried to put the file on the `.gitignore` file but no luck yet. It is just really annoying.

File: public/index.html (hunk 2 above)
Comment (Atos20) [reply public/index.html]: To me, this is just some instruction by React but we don't need them. This file is what displays our entire React application and translated into what is called the DOM.

File: src/_variables.css (hunk 3 above)
Comment (Atos20) [reply src/_variables.css:7]: That is the idea, but I haven't been able to make use of the variable the way you just described it.

File: src/components/App/App.tsx (hunk 4 above)
Comment (Atos20) [reply src/components/App/App.tsx:6]: Exactly!, this App React component is the one that fires the API call to populate the survey and holding on to the Survey's current state, from here we will send you the hash you need to get back the report.

File: src/components/Journey/Journey.tsx [hunk 10]
```diff
@@ -1,21 +1,41 @@
...
+            <h1 className='title-2'>Dream Home</h1>
+          </div>
+        </div>
+        
+        <div className='img-box'>
+          <img src={bkg_img} alt="" className='journey-img'/>
+        </div>
+
+        <div className="btn-container">
+          <button className="btn">Start</button>
```
Comment (BlakeDonn) [inline src/components/Journey/Journey.tsx:32]: I think the only thing we will need to add here is just having this button be a link, or some other form of navigation to route to the next page.

**Review by BlakeDonn — APPROVED** [review]

File: src/components/Home/Home.tsx (hunk 6 above)
Comment (timkeresey) [reply src/components/Home/Home.tsx:2]: This is the whole component. React essentially creates components that are functions containing an element's functionality, and returns an HTML element.  the element in the return block is actually in JSX which is JS but emulates HTML and compiles as HTML.

File: src/components/Journey/Journey.tsx (hunk 10 above)
Comment (timkeresey) [reply src/components/Journey/Journey.tsx:32]: I think a <Link> wrapper is a good way to do that.

**Review by timkeresey — APPROVED** [review]

**Pull request merged**